fem/estimators.cpp

```diff
@@ -50,4 +50,21 @@ void L2ZienkiewiczZhuEstimator::ComputeEstimates()
 
 #endif // MFEM_USE_MPI
 
+void LpErrorEstimator::ComputeEstimates()
+{
+   MFEM_VERIFY(coef != NULL || vcoef != NULL,
+               "LpErrorEstimator has no coefficient!  Call SetCoef first.");
+
+   error_estimates.SetSize(sol->FESpace()->GetMesh()->GetNE());
+   if (coef)
+   {
+      sol->ComputeElementLpErrors(local_norm_p, *coef, error_estimates);
+   }
+   else
+   {
+      sol->ComputeElementLpErrors(local_norm_p, *vcoef, error_estimates);
```

Comment:
What if both `coef` and `vcoef` are NULL (first constructor)?

Reply:
Doesn't the `MFEM_VERIFY` handle this case.  Did I get the logic wrong?

Reply:
Oh I overlooked that, sorry...

Reply:
No, worries.  It's always best to double check such things.